api/app/controllers/spree/api/price_lists_controller.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+module Spree
```

Comment:
the api was still up for discussion, please provide copy and pasted json of your proposal. Please do not extend the prediscussed and documented frame of the issue. If there's written TBD either provide a suggestion inside the issue in written from BEFORE INTITIATING THE WORK or wait on a proposal